utopia-api/src/primitives/positionable.tsx

```diff
@@ -11,6 +11,10 @@ export interface PositionableProps
   wrappedComponent: React.JSXElementConstructor<any>
 }
 
+const tempFix = (props: any) => {
+  return <div />
+}
```

Comment:
I have a feeling this temporary fix isn't going to remain temporary, so please can you rename this accordingly.